Himanshu_Kumar/project1-(calculator)/assets/css/cal_css.css

```diff
@@ -0,0 +1,46 @@
+.cal-div {
+  background-color: rgba(128, 128, 128, 0.329);
+  box-shadow: 10px 10px 20px rgba(128, 128, 128, 0.781);
```

Comment:
@Hhumour can you explain box-shadow property

Reply:
for the shadow to shift from origin, I have given three parameters. (10px,10px) is to shift from left and top respectively. 20px is for how much you want to  pixelette the sahdow.